.gitignore

```diff
@@ -5,4 +5,6 @@ TODO
 *.install
 data/
 packages
-.~*
\ No newline at end of file
+.~*
+_opam/
+_build/
```

Comment:
I've added these 2 lines for developer comfort (opam switch and dune build, respectively).